src/Bigcommerce/Api/Client.php

```diff
@@ -1607,4 +1596,297 @@ public static function deletePage($pageId)
+    }
+
+    /**
+     * Add a address to the customer's address book.
+     *
+     * @param $customerID
+     * @param $object
+     * @return mixed
+     */
+    public static function createCustomerAddress($customerID, $object)
+    {
+        return self::createResource('/customers/' . $customerID . '/addresses', $object);
+    }
+
+    /**
+     * Create a product rule
+     *
+     * @param $productID
+     * @param $object
+     * @return mixed
+     */
+    public static function createProductRule($productID, $object)
+    {
+        return self::createResource('/products/' . $productID . '/rules', $object);
+    }
+
+    /**
+     * Create a customer group.
+     *
+     * @param $object
+     * @return mixed
+     */
+    public static function createCustomerGroup($object)
+    {
+        return self::createResource('/customer_groups', $object);
+    }
+
+    /**
+     * Get list of customer groups
+     *
+     * @return mixed
+     */
+    public static function getCustomerGroups()
+    {
+        return self::getCollection('/customer_groups');
+    }
+
+    /**
+     * Delete a customer group
+     *
+     * @param $customerGroupId
+     * @return mixed
+     */
+    public static function deleteCustomerGroup($customerGroupId)
+    {
+        return self::deleteResource('/customer_groups/' . $customerGroupId);
+    }
+
+    /**
+     * Delete all customers
+     *
+     * @return mixed
+     */
+    public static function deleteAllCustomers()
+    {
+        return self::deleteResource('/customers');
+    }
+
+    /**
+     * Delete all options
+     *
+     * @return mixed
+     */
+    public static function deleteAllOptions()
+    {
+        return self::deleteResource('/options');
+    }
+
+    /**
+     * Return the collection of all option values for a given option.
+     *
+     * @param $productId
+     * @return mixed
+     */
+    public static function getProductOptions($productId)
+    {
+        return self::getCollection('/products/' . $productId . '/options');
+    }
+
+    /**
+     * Return the collection of all option values for a given option.
+     *
+     * @param $productId
+     * @param $productOptionId
+     * @return mixed
+     */
+    public static function getProductOption($productId, $productOptionId)
+    {
+        return self::getResource('/products/' . $productId . '/options/' . $productOptionId);
+    }
+
+    /**
+     * Return the collection of all option values for a given option.
+     *
+     * @param $productId
+     * @param $productRuleId
+     * @return mixed
+     */
+    public static function getProductRule($productId, $productRuleId)
+    {
+        return self::getResource('/products/' . $productId . '/rules/' . $productRuleId);
+    }
+
+    /**
+     * Return the option value object that was created.
+     *
+     * @param $object
+     * @param int $option_id
+     * @return mixed
+     */
+    public static function createOptionValue($option_id, $object)
```

Comment:
we're using camel case for the rest of these params, can we keep it consistent here & below?